[DNS] Audit and replace external broken link (#12740)
cloudflare/cloudflare-docs: content/dns/dnssec/multi-signer-dnssec/about.md

# About multi-signer DNSSEC

{{<Aside type="note">}}

> This is a simplified explanation to give you context and clarify what is involved in a [multi-signer DNSSEC setup](/dns/dnssec/multi-signer-dnssec/setup/). For technical details refer to [RFC 8901](https://www.rfc-editor.org/rfc/rfc8901.html). To read more about DNSSEC, refer to [How DNSSEC works](https://blog.cloudflare.com/dnssec-an-introduction/).

Reviewer: ```suggestion
This is a simplified explanation to give you context and clarify what is involved in a [multi-signer DNSSEC setup](/dns/dnssec/multi-signer-dnssec/setup/). For technical details refer to [RFC 8901](https://www.rfc-editor.org/rfc/rfc8901.html). To read more about DNSSEC, refer to our blog post [DNSSEC: An Introduction](https://blog.cloudflare.com/dnssec-an-introduction/).
```
